jumanji/environments/packing/bin_pack/env_test.py

```diff
@@ -503,6 +535,259 @@ def test_bin_pack__optimal_policy_random_instance(
                 state, timestep = step_fn(state, action)
                 # assert not timestep.extras["invalid_action"]
                 assert not jnp.any(timestep.extras["invalid_ems_from_env"])
-            assert jnp.array_equal(state.items_placed[0], solution.items_placed)
+            assert jnp.array_equal(state.items_placed, solution.items_placed)
             assert round(timestep.extras["volume_utilization"]) == 1
             assert timestep.extras["ratio_packed_items"] == 1
+
+
+class TestExtendedBinPackRotationValue:
+
+    # TO-DO:
```

Comment:
Are these still todo or can this comment be removed?

Could you add a docstring?

Reply:
Done ✅